include/libopencm3/sam/4l/pm.h

```diff
@@ -191,4 +184,19 @@ enum mck_src {
 	MCK_SRC_RC1M,
 };
 
+enum cksel {
+	CKSEL_CPU = 0,
+	CKSEL_PBA = 2,
+	CKSEL_PBB,
+	CKSEL_PBC,
+	CKSEL_PBD,
```

Comment:
You've not got CKSEL and MCK plus all of PM_ from this module.  Please consider whether you want all of that in the global namespace.